cmd/govim/testdata/scenario_default/quickfix_from_other.txt

```diff
@@ -0,0 +1,95 @@
+# Test that the quickfix window isn't updated by govim if it isn't populated by govim
+
+vim ex 'e main.go'
+vim ex 'call setqflist([{\"filename\":\"foo\"}, {\"filename\":\"bar\"}, {\"filename\":\"baz\"}], \"r\")'
+vimexprwait errors.other GOVIMTest_getqflist()
+# Duplicate error, the quickfix should remain because it hasn't been populated by govim
```

Comment:
If you put a blank line before this comment, [`testscript`](https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript) will treat it as a section. In the verbose output from `testscript`, sections that complete "successfully" are collapse under the "heading" of the comment that precedes the contiguous lines of that section.

Reply:
done 35c475d